vslib/SwitchBCM81724.cpp

```diff
@@ -273,6 +278,7 @@ sai_status_t SwitchBCM81724::refresh_read_only(
                  */
 
             case SAI_PORT_ATTR_OPER_STATUS:
+            case SAI_PORT_ATTR_PORT_SERDES_ID:
```

Comment:
SAI_PORT_ATTR_PORT_SERDES_ID is readonly attribute, and normally it should be recalculated each time is queried but if it's null, then is ok to jus return here

Reply:
@kcudnik Do I need to fix something here? Other than here, it looks like all checks haven been pass.

Reply:
you are fine